api/integrations/google/oauth.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import datetime
+import httplib2
+import os
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from oauth2client import client, tools
+from oauth2client.file import Storage
+
+# Before running this, make sure to register a new
+# project in the Google Developers Console
+# (https://console.developers.google.com/start)
+# and download the "client secret" JSON file.
+#
+# You'll have to specify its location with --secret-path.
+
+# This parent argparser is used because it already contains
+# the arguments that Google's Client Library method "tools.run_flow"
+# supports.
```

Comment:
Agreed on calendar, but I still think the `oauth.py` should have some information on what the `scopes` argument is and how to use it - this file is now not self-explanatory.